### PR TITLE
added the style and html elements to seller page

### LINKED ISSUE #8: seller_html_css
we need to add a form on the general  html CSS added for the buyer UI the form should take the (name ,price ,etc )for the product

### PR DESCRIPTION
**Pull request by Sondos-Alalami**
added the style and html elements to seller page

### CODE REVIEW

File: seller.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    </nav>
+    
+    <button id="addNewProductBtn">Add new product</button>
+    
+    <form  action="#" id="popUpForm">
+        <div class="formContainer">
+            <label class="item">Product name</label>
+            <input type="text" class="item" id="productName">
+            <label class="item">Category</label>
+            <input type="text" class="item" id="productCategory">
```
Comment (zeinjendeya) [inline seller.html]: this should be a list

File: styles/seller.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
+*{
+    box-sizing: border-box;
+}
+
```
Comment (zeinjendeya) [inline styles/seller.css:4]: add rulset for html with font size

**Review by zeinjendeya — CHANGES_REQUESTED** [review]

File: seller.html (hunk 1 above)
Comment (Sondos-Alalami) [reply seller.html]: why ? we can take each input value and store it in its place in the object

**Review by Osama-AlAttar — APPROVED** [review]

File: seller.html (hunk 1 above)
Comment (zeinjendeya) [reply seller.html]: as written in the project description the seller should choose the category from a drop list
if you want it this way and have further plans with it, I understand.

**Review by zeinjendeya — APPROVED** [review]

**Review by AlaaShurrab — APPROVED** [review]

**Pull request merged**